src/pages/components/notification/usage.mdx

```diff
@@ -4,79 +4,166 @@ description: Notifications are messages that communicate information to the user
+
+### Overflow
+
+Notification messages should be kept short. For toast notifications, the timestamp is optional and can be removed if a third line of content is needed.
+
+If a toast or inline notification requires a message of longer than three lines, include a short message with a “View more” link that takes the user to a view of the full notification message. This can be either a full page with more details or a modal.
+
+## Content
+
+Notifications provide limited space for content, and therefore the content must be concise. A user should be able to quickly scan the notification, be apprised of the situation, and know what to do next.
+
+### Title
+
+- The title should be short and descriptive, explaining the most important piece of information.
+- When possible, communicate the main message using just the title.
+- Use a period only if the title is a full sentence.
 
-Inline notifications appear near its related item. In [forms](/components/form/code), we recommend placing the inline notification at the bottom of the form, right before the submission buttons. Depending on the context of the page, inline notifications can appear above the content as well.
+### Body content
+
+- The body of the notification should be no more than two lines.
+- Be concise and avoid repeating or paraphrasing the title.
+- Explain how to resolve the issue by including any troubleshooting actions or next steps. You can include links within the notification body that redirect the user to next steps.
+
+### Action
+
+- Keep labels consice and clearly indicate the action the user can take.
+- Limit action labels to two words or less.
+
+### Timestamp
+
+- The timestamp reflects the time the toast notification was sent. Timestamps should be localized to the user's timezone or preferences.
+
+For further content guidance, see Carbon's [content guidelines](/guidelines/content/general).
+
+## Modifiers
+
+### High and low contrast
+
+Carbon supports high and low constrast style notifications. High-contrast notifications are best for critical messaginging while low-contrast notifications are less visually disruptive for users.
+
+It's' up to the product team to decide which notification style to use in their product. Inline and toast notifications can use different styles but you should never mix styles within each notification type. When in doubt, use low-contrast notifications.
 
 <Row>
 <Column colLg={8}>
 
-![Form example with inline notification](images/notification-usage-3.png)
+![High and low contrast inline notifications](images/notification-usage-4.png)
 
 </Column>
 </Row>
+
+### Action
+
+**Inline notification**
+
+Inline notifications have an optional ghost button action that is right aligned. This button should allow users to take further action on the notification.
```

Comment:
> Inline notifications have an optional ghost button action that is **right aligned**

What would be the best way to talk about direction if we do left-to-right and right-to-left? Or is this is saying that the action is always on the right-hand side?

Reply:
We dont really address this topic anywhere and at the moment we dont really have a strategy for it. This is something we should talk about though.

Reply:
Speaking of the action button placement though this week we are adding a small width behavior change to it. So it sometime also on the bottom.
https://github.com/carbon-design-system/carbon/pull/5763